.github/dependabot.yml

```
@@ -1,17 +1,25 @@
 version: 2
 updates:
-- package-ecosystem: "gomod"
-  directory: "/"
-  schedule:
-    interval: "weekly"
-    day: "monday"
-    time: "08:00"
-    timezone: "Europe/Oslo"
-    
-- package-ecosystem: "github-actions"
-  directory: "/"
-  schedule:
-    interval: "weekly"
-    day: "monday"
-    time: "08:00"
-    timezone: "Europe/Oslo"
\ No newline at end of file
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "08:00"
+      timezone: "Europe/Oslo"
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "08:00"
+      timezone: "Europe/Oslo"
+
+  - package-ecosystem: "terraform"
+    directory: "/"
```

Her må nok directory stå til deployment
```suggestion
    directory: "deployment"
```